app/shared/resource-calendar/ResourceCalendar.js

```diff
@@ -59,59 +64,77 @@ export class UnconnectedResourceCalendar extends Component {
 
     return (
       <div className="app-ResourceCalendar">
-        <FormGroup onClick={this.showOverlay}>
-          <InputGroup>
-            <InputGroup.Addon>
-              <img alt="" className="app-ResourceCalendar__icon" src={iconCalendar} />
-            </InputGroup.Addon>
-            <FormControl disabled type="text" value={selectedDateText} />
-            <InputGroup.Addon>
-              <Glyphicon glyph="triangle-bottom" />
-            </InputGroup.Addon>
-          </InputGroup>
-        </FormGroup>
-        <Overlay
-          container={this}
-          onHide={this.hideOverlay}
-          placement="bottom"
-          rootClose
-          show={this.state.visible}
-        >
-          <ResourceCalendarOverlay onHide={this.hideOverlay}>
-            <DayPicker
-              disabledDays={this.disableDays}
-              enableOutsideDays
-              initialMonth={new Date(selectedDate)}
-              locale={currentLanguage}
-              localeUtils={MomentLocaleUtils}
-              modifiers={{
-                available: (day) => {
-                  const dayDate = day.toISOString().substring(0, 10);
-                  return availability[dayDate] && availability[dayDate].percentage >= 80;
-                },
-                busy: (day) => {
-                  const dayDate = day.toISOString().substring(0, 10);
-                  return (
-                    availability[dayDate] &&
-                    availability[dayDate].percentage < 80 &&
-                    availability[dayDate].percentage > 0
-                  );
-                },
-                booked: (day) => {
-                  const dayDate = day.toISOString().substring(0, 10);
-                  return availability[dayDate] && availability[dayDate].percentage === 0;
-                },
-              }}
-              onDayClick={this.handleDateChange}
-              selectedDays={selectedDay}
-            />
-            <div className="calendar-legend">
-              <span className="free">{t('ReservationCalendarPickerLegend.free')}</span>
-              <span className="busy">{t('ReservationCalendarPickerLegend.busy')}</span>
-              <span className="booked">{t('ReservationCalendarPickerLegend.booked')}</span>
-            </div>
-          </ResourceCalendarOverlay>
-        </Overlay>
+        <button
+          className="app-ResourceCalendar__week-button app-ResourceCalendar__week-button--prev"
+          onClick={() => this.handleDateChange(
+            moment(selectedDay).subtract(1, 'w').toDate()
+          )
+          }
```

Comment:
Formatting.. should be `)}` instead of `)\n}`

Reply:
@rBoost : That's what i hate about prettier..

Reply:
yep. Probably the latter one.